frontend/corrections.py

```diff
@@ -1,9 +1,8 @@
 import time
-from datetime import datetime, timedelta
-from decimal import Decimal
-
 import boto3
 import pymysql.cursors
+from datetime import datetime, timedelta

```

Comment:
please use pylance and python extension of visual studio code to work with these files and let it sort the imports

Reply:
So I noticed I was using Pylance but it didn't show me an errors or warnings with the way I had sorted the imports, for safer side reverted to as it was.